aten/src/ATen/native/cuda/UnaryGammaKernels.cu

```diff
@@ -11,7 +11,7 @@
 namespace at { namespace native {
 
 void digamma_kernel_cuda(TensorIterator& iter) {
-  AT_DISPATCH_FLOATING_TYPES_AND_HALF(iter.dtype(), "digamma_cuda", [&]() {
+  AT_DISPATCH_FLOATING_TYPES_AND_HALF(iter.common_dtype(), "digamma_cuda", [&]() {
```

Comment:
Does the C++ dispatch also need an update like this?

Reply:
Updated Thanks!